README.md

```diff
@@ -77,6 +77,21 @@ With valgrind, you might need to increase the max stack size:
 
     $ valgrind --max-stackframe=2500000 ./exhaustive_tests
 
+Test coverage
+-----------
+
+To create a test coverage report, configure with `--enable-coverage`:
+
+    $ ./configure --enable-coverage=yes
+
+Run the tests:
+
+    $ make check
+
+To create a HTML report, `gcovr` is recommended, as it visualizes the coverage of all branches:
+
+    $ gcovr -r . --html --html-details -o coverage.html
```

Comment:
Since we're on the command line anyway, I think `gcovr --exclude 'src/bench*' --print-summary` is a better option.

Reply:
Added, but also kept the HTML anyways, since I mainly use this one to check which lines are not covered.